### DISCUSSION TITLE
US hardware list - looking for multi outlet plugs

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#18574] [Q&A]
It's a bit frustrating to buy things off Amazon and hope they will convert and I'm wondering if there are any known pre-flashed US power strips that may not be listed in the hardware list?

**Comment by barbudor** [comment]
Your guess is as good as ours
If it's not in the template repository, we have no way to know.
It's up to everyone goodwill to report good and bad devices in the repository

**Reply by dac2020** [reply]
I get that but it's worth asking in case some have devices running they haven't shared yet.

**Reply by sfromis** [reply]
By now, the default assumption has to be that random devices are not "Tasmota-compatible", unless you have a specific reason to believe so. If you are lucky, it is not too hard to replace the Wifi module by one which works with Tasmota.

One alternate strategy would be to not depend on flashing Tasmota on end devices, instead using Zigbee for local-only IoT, including Tasmota integration via a coordinator running Tasmota. While this does not give you full Tasmota flexibility of detail control, most of the time you may not need that.

**Reply by dac2020** [reply]
I use a fair number of tasmota commands so would like to retain as many functions as possible. 
All of my devices are remote to me, a few are local.

Like I said, just would love to find pre-installed devices so I ask now and then :).
